README.md

```diff
@@ -1,6 +1,10 @@
 # PyoMyo
 Python module for the Thalmic Labs Myo armband. 
 
+Adding additional information here.
+
```

Comment:
Max this is awesome and elegant -- nice use of variables

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Unreadable, Not secure, Inefficient, Elegant